apps/backoffice-v2/src/domains/workflows/fetchers.ts

```diff
@@ -178,7 +179,9 @@ export const fetchWorkflowDecision = async ({
     endpoint: `workflows/${workflowId}/decision/${documentId}`,
     method: Method.PATCH,
     body,
-    schema: WorkflowByIdSchema,
+    schema: WorkflowByIdSchema.pick({
```

Comment:
Currently it's a substantial amount of effort to format the response workflow the same way as in `getWorkflowByIdWithRelations`, the way this fetcher is used `context` is the only prop needed, and doing so resolves errors on revision/approve for a task.